e2-studio-star-rx72n-firmware/tests/mocks/mock_adc_hal.c

```
@@ -261,18 +414,61 @@ rx_err_t adc_read(adc_unit_t unit, adc_channel_t channel, uint16_t* value)
+ * uint32_t mv = 0U;
+ * rx_err_t err = adc_read_voltage_mv(k_adc_unit_0, k_adc_channel_0,
+ *                                    k_adc_resolution_12bit, &mv);
+ * TEST_ASSERT_EQUAL(k_rx_ok, err);
+ * TEST_ASSERT_UINT32_WITHIN(2U, 1650U, mv);
+ * @endcode
+ *
+ * @see adc_read() Raw-count read used internally
+ * @see adc_init() Must be called before this function
+ * @since Version 1.0.0
+ */
 rx_err_t adc_read_voltage_mv(adc_unit_t       unit,
```

Clang (14.0.6)

[warning] 3 adjacent parameters of 'adc_read_voltage_mv' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)